---
title: "HPRC PUBLICATIONS"
date: 2020-06-23T12:33:45+10:00
draft: false
featured: true
weight: 1
description: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec accumsan dapibus purus, vel maximus erat facilisis vel.
---

#### There are several HPRC publications in development at this time. Please check back for updates and follow us on Twitter @[HumanPangenome](https://twitter.com/humanpangenome?lang=en).  Below is a selection of publications from the HPRC membership.  


[The Human Pangenome Project: a global resource to map genomic diversity.](https://www.nature.com/articles/s41586-022-04601-8)  
*Wang T., Antonacci-Fulton L., Howe K., et al., 2022-04-20*

[Pangenome-based genome inference allows efficient and accurate genotyping across a wide spectrum of variant classes.](https://www.nature.com/articles/s41588-022-01043-w)  
*Ebler J, Ebert P, Clarke WE, Rausch T, Audano PA, Houwaart T, Mao Y, Korbel JO, Eichler EE, Zody MC, Dilthey AT, Marschall T., 2022-04-11*

[Concerted modification of nucleotides at functional centers of the ribosome revealed by single-molecule RNA modification profiling.](https://elifesciences.org/articles/76562)  
*Bailey AD, Talkish J, Ding H, Igel H, Duran A, Mantripragada S, Paten B, Ares M.Elife, 2022-04-06*

[Segmental duplications and their variation in a complete human genome.](https://www.science.org/doi/10.1126/science.abj6965)  
*Vollger MR, Guitart X, Dishuck PC, et al., 2022-04-01*

[StainedGlass: Interactive visualization of massive tandem repeat structures with identity heatmaps.](https://pubmed.ncbi.nlm.nih.gov/35020798/)  
*Vollger MR, Kerpedjiev P, Phillippy AM, Eichler EE., 2022-01-10*

[The UCSC Genome Browser database: 2022 update.](https://pubmed.ncbi.nlm.nih.gov/34718705/)  
*Lee BT, Barber GP, Benet-Pag√®s A, et al., 2022-01-07*

[Profiling variable-number tandem repeat variation across populations using repeat-pangenome graphs](https://doi.org/10.1038/s41467-021-24378-0)  
*Tsung-Yu Lu, Human Genome Structural Variation Consortium, Mark Chaisson., 2021-07-12*

[Targeted long-read sequencing identifies missing disease-causing variation](https://doi.org/10.1016/j.ajhg.2021.06.006)  
*Miller, D., Sulovari, A., Wang, T., et al., 2021-07-02*

[lra: A long read aligner for sequences and contigs](https://doi.org/10.1371/journal.pcbi.1009078)  
*Jingwen Ren, Mark Chaisson., 2021-06-21*

[The qBED track: a novel genome browser visualization for point processes](https://doi.org/10.1093/bioinformatics/btaa771)  
*Moudgil, A., Li, D., Hsu, S., et al., 2021-05-23*

[Common DNA methylation dynamics in endometriod adenocarcinoma and glioblastoma suggest universal epigenomic alterations in tumorigenesis](https://doi.org/10.1038/s42003-021-02094-1)  
*Karlow, J., Miao, B., Xing, X., et al., 2021-05-21*

[A high-quality bonobo genome refines the analysis of hominid evolution](https://doi.org/10.1038/s41586-021-03519-x)  
*Mao, Y., Catacchio, C., Hillier, L., et al., 2021-05-05*

[The Need for a Human Pangenome Reference Sequence](https://doi.org/10.1146/annurev-genom-120120-081921)  
*Miga, K., Wang, T., 2021-04-30*

[Human placental cytotrophoblast epigenome dynamics over gestation and alterations in placental disease](https://doi.org/10.1016/j.devcel.2021.04.001)  
*Zhang, B., Kim, B., Elliot, G., et al., 2021-04-20*

[Profiling SARS-CoV-2 mutation fingerprints that range from the viral pangenome to individual infection quasispecies](https://doi.org/10.1186/s13073-021-00882-2)  
*Lau, B., Pavlichin, D., Hooker, A., et al., 2021-04-19*

[The structure, function and evolution of a complete human chromosome 8](https://doi.org/10.1038/s41586-021-03420-7)  
*Logsdon, G., Vollger, M., Hsieh, P., et al., 2021-04-07*

[Haplotype-resolved diverse human genomes and integrated analysis of structural variation](https://doi.org/10.1126/science.abf7117)  
*Ebert, P., Audano, P., Zhu, Q., et al., 2021-02-25*

[Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm](https://doi.org/10.1038/s41592-020-01056-5)  
*Cheng, H., Concepcion, G., Feng, X., et al., 2021-02-01*

[Fully phased human genome assembly without parental data using single-cell strand sequencing and long reads](https://doi.org/10.1038/s41587-020-0719-5)  
*Porubsky, D., et al., 2020-12-07*

[Chromosome-scale, haplotype-resolved assembly of human genomes](https://doi.org/10.1038/s41587-020-0711-0)  
*Garg, S., Fungtammasan, A., Carroll, A., et al., 2020-12-07*

[A map of cis-regulatory elements and 3D genome structures in zebrafish](https://doi.org/10.1038/s41586-020-2962-9)  
*Yang, H., Luan, Y., Liu, T., et al., 2020-11-25*

[The design and construction of reference pangenome graphs with minigraph](https://doi.org/10.1186/s13059-020-02168-z)  
*Li, H., Feng, X., Chu, C., 2020-10-16*

[Cell-type-specific 3D epigenomes in the developing human cortex](https://doi.org/10.1038/s41586-020-2825-4)  
*Song, M., Pebworth, M., Yang, X., et al., 2020-10-14*

[Tissue-specific usage of transposable element-derived promoters in mouse development](https://doi.org/10.1186/s13059-020-02164-3)  
*Miao, B., Fa, S., Lyu, C., et al., 2020-09-28*

[Telomere-to-telomere assembly of a complete human X chromosome](https://doi.org/10.1038/s41586-020-2547-7)  
*Miga, K., Koren, S., Rhie, A. et al., 2020-07-14*

[Epigenomics and genotype-phenotype association analyses reveal conserved genetic architecture of complex traits in cattle and human](https://doi.org/10.1186/s12915-020-00792-6)  
*Liu, S., Yu, Y., Zhang, S., et al., 2020-07-03*

[The design and construction of reference pangenome graphs](https://arxiv.org/abs/2003.06079)  
*Li, H., Feng, X., Chu, C., 2020-03-13*

[Genotyping structural variants in pangenome graphs using the vg toolkit](https://doi.org/10.1186/s13059-020-1941-7)  
*Hickey, G., Heller, D., Monlong, J., et al., 2020-02-12*

[A fully phased accurate assembly of an individual human genome](https://doi.org/10.1101/855049)  
*Porubsky, D., Ebert, P., Audano, P., et al., 2019-11-26*

[Efficient de novo assembly of eleven human genomes using PromethION sequencing and a novel nanopore toolkit](https://doi.org/10.1101/715722)  
*Shafin, K., Pesout, T., Lorig-Roach, R., et al., 2019-07-26*

[Progressive alignment with Cactus: a multiple-genome aligner for the thousand-genome era](https://doi.org/10.1101/730531)  
*Armstrong, J., Hickey, G., Diekhans, M., et al. , 2019-10-09*




